src/main/java/uk/gov/hmcts/reform/slc/logging/AppDependency.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.slc.logging;
+
+final class AppDependency {
+
+    /**
+     * Service bus message receiver dependency.
+     * Used to track dependency of how long in millis did it take to...
+     */
+    static final String MESSAGE_RECEIVER_DEPENDENCY     = "MessageReceiver";
```

Comment:
is this service bus basically?

Reply:
Yep. Ah. Get it. Name dependency with actual name:
- ServiceBus
- FtpClient
- PdfGenerator

Right?